server/src/test/java/org/apache/iotdb/db/integration/IoTDBGroupByFillIT.java

```diff
@@ -316,6 +317,7 @@ public void previousUntilLastTest1() {
     try (Connection connection = DriverManager.
             getConnection("jdbc:iotdb://127.0.0.1:6667/", "root", "root");
          Statement statement = connection.createStatement()) {
+      LastCacheManager.getInstance().clear();
```

Comment:
I would suggest moving this into `clearEnv` in `EnvironmentUtils` so that other people will not forget to do the cleaning when they add new tests.

Reply:
Fixed